sandbox/docker/master/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
+
+WORKDIR /tmp
+COPY ./requirements.txt /tmp/
+
+RUN apt-get update; \
```

Comment:
We have a syntax error here. Could you remove `;` from this line and the next line?